README.md

```diff
@@ -59,7 +59,7 @@ weather === "sunny";
 
 What are the values of these expressions?  Explain your answers.
 
-<!-- Replace this comment with your answer -->
+in line 2 the value is "sunny", in line 3 the value is "true". in line two, you're defining a variable "weather" as "sunny", but in line three you're setting up a boolean for whether var weather is "sunny" (=true) or not (=false).
```

Comment:
Very nice! Just don't forget about that first line. `var weather;` is also an expression, and it evaluates to `undefined`.